front/src/modules/companies/hooks/useOpenContextMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import { getOperationName } from '@apollo/client/utilities';
+import { IconCheckbox, IconNotes, IconTrash } from '@tabler/icons-react';
+import { useRecoilState, useRecoilValue, useSetRecoilState } from 'recoil';
+
+import { useOpenCreateActivityDrawerForSelectedRowIds } from '@/activities/hooks/useOpenCreateActivityDrawerForSelectedRowIds';
+import { GET_PIPELINES } from '@/pipeline/queries';
+import { ContextMenuEntry } from '@/ui/context-menu/components/ContextMenuEntry';
+import { useResetTableRowSelection } from '@/ui/table/hooks/useResetTableRowSelection';
+import { contextMenuEntriesState } from '@/ui/table/states/ContextMenuEntriesState';
```

Comment:
let's move this to contextMenu folder. This way, context menu does not have a dependency on table

ContextMenu (resp. ActionBar) doesn't know about table
Table doesn't know about ContextManu (resp. ActionBar)
Company module makes the glue between them

This enforce clear dependency direction, which has a huge value regarding refactoring. If we want to go one step further, we can even have ContextMenu (resp. ActionBar) exposing a hook whose functions will themselves modify the ContextMenu state and will be ContextMenu API so only ContextMenu is touching its own internal stuff
I don't think it's necessary at this stage to enforce an API through hooks but it will likely come in the future as the project gets more mature and things more stable